app/modules/vm/index.coffee

```diff
@@ -162,6 +163,8 @@ module.exports = angular.module 'xoWebApp.vm', [
         oVdi = get oVbd.VDI
         continue unless oVdi
         VDIs.push oVdi if oVdi and not oVbd.is_cd_drive
+        if (isNumber(oVdi.size))
+          oVdi.size = bytesToSizeFilter(oVdi.size)
```

Comment:
I don't like to alter a global object, this can cause unexpected behaviour throughout the app.

But we may already do that in xo-web, this will be protected against in 5.0.